app/models/user.rb

```diff
@@ -1,11 +1,20 @@
 # frozen_string_literal: true
 class User < ApplicationRecord
-  # Include default devise modules. Others available are:
-  # :confirmable, :lockable, :timeoutable, :trackable and :omniauthable
+  ADMIN = 'admin'
+  ROLES = [ADMIN].freeze
 
   ## TO-DO - add custom validation for email address ???
   validates :name, presence: true
   validates :email, uniqueness: true
-  devise :database_authenticatable, :registerable,
-         :recoverable, :rememberable, :validatable, authentication_keys: [:email]
+  validates :role, inclusion: { in: ROLES }, allow_nil: true
```

Comment:
This line validates that the user's role is valid. I've written all this to make it super easy to add more roles later (e.g. you can just add a new string to the `ROLES` array defined above.